tools/ivwpy/regression/imagecompare.py

```diff
@@ -44,11 +44,28 @@ def __init__(self, testImage, refImage, enhance = 10):
         self.numberOfDifferentPixels = None
         self.maxDifference = None
 
+        channels = len(self.testImage.getbands());
+
+
+        # Inviwo don't save PNG using a pallete, if test image is a pallete we
+        # need to convert it to RGB or RGBA
+        if self.refImage.mode == 'P':
```

Comment:
what about 1/2 channels?

Reply:
In a PNG image a palette is always representing RGB colors. Alpha can be added as an additional channel. 
Anyway, I changed this a bit, see a9bb185b115f5ba319cf72b1e957ad9dcf88de32